addons/website/static/src/snippets/s_dynamic_snippet/000.js

```diff
@@ -159,7 +159,7 @@ const DynamicSnippet = publicWidget.Widget.extend({
         return {
             chunkSize: chunkSize,
             data: this.data,
-            uniqueId: this.uniqueId,
+            id: this.uniqueId,
```

Comment:
`id` is pretty bad to grep in the code, could you use something else?
`identifier`?
And probably also rename the `uniqueId` property of `this` as there is only one occurence?
```suggestion
            identifier: this.identifier,
```
Could also go with:
- `unique_id` but with the risk of someone camelcasing it later and back to problem
- `idUnique`
- ..